README.md

```diff
@@ -45,7 +45,7 @@ auth).
 
 You will additionally need to specify the OAuth callback URL, the client ID, and
 the client secret (you should have gotten these when you created your OAuth app
-on GitHub). For example, if these values are in the environment variables
+on GitHub). For example, if these values are in the environment variables ([`/etc/environment` in Ubuntu](https://help.ubuntu.com/community/EnvironmentVariables#System-wide_environment_variables))
```

Comment:
Credentials like these shouldn't actually go in system-wide environment variables, because that would expose information to other users on the system. The Ubuntu page is good, but can we remove the reference to system-wide settings / `/etc/environment`?